dask_kubernetes/operator/controller/controller.py

```diff
@@ -410,18 +472,18 @@ async def retire_workers(
     )
     # Dask version mismatches between the operator and scheduler may cause this to fail in any number of unexpected ways
     with suppress(Exception):
-        comm_address = await get_scheduler_address(
+        comm_address = await get_scheduler_address(  # type: ignore[no-untyped-call]
```

Comment:
There are a lot of these ignore throughout. Perhaps it makes sense to ignore this globally via the project config?

Reply:
I'm personally more on the "the more explicit and correct, the better"-side (because ignoring it globally *could* potentially mask bugs, depending on which error you want to ignore) but I'm happy to use the way that works for everybody. Note that these untyped calls would actually vanish as soon as the entire repo would be properly typed.

Reply:
I think the thing I would say here is that Dask strives to be a project that is welcoming to novice developers. My concern with type hinting in general is it can be very daunting for junior developers, even if it helps them avoid shooting their foot off.

Removing inline ignores I feel is a good way to remove some visual clutter that may be daunting for some folks. So even though it may mask a bug here and there I would prefer a global ignore.

Reply:
Sure, if you'd prefer a global ignore for `no-untyped-call` & `import-untyped`, we can integrate this into the `pyproject.toml`. I'll do it when I find time for it.